frontend/.eslintignore

```diff
@@ -0,0 +1,9 @@
+# Ignore 
+
+node_modules/
+frontend/node_modules/
+.*
+Gemfile 
+config.ru 
+Rakefile 
+README.md 
```

Comment:
This is not only "Eslint" related. You're also ignoring other files. Plus, why ignore changes to README?